build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -0,0 +1,142 @@
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '10.15.3'
+
+  - task: geeklearningio.gl-vsts-tasks-yarn.yarn-installer-task.YarnInstaller@3
+    inputs:
+      versionSpec: '1.x'
+
+  - task: 1ESLighthouseEng.PipelineArtifactCaching.RestoreCacheV1.RestoreCache@1
+    inputs:
+      keyfile: '.yarnrc, remote/.yarnrc, **/yarn.lock, !**/node_modules/**/yarn.lock, !**/.*/**/yarn.lock, !samples/**/yarn.lock'
+      targetfolder: '**/node_modules, !**/node_modules/**/node_modules, !samples/**/node_modules'
+      vstsFeed: 'BuildCache'
+
+  - script: |
+      set -e
+      CHILD_CONCURRENCY=1 yarn --frozen-lockfile
+    displayName: Install dependencies
+    condition: and(succeeded(), ne(variables['CacheRestored'], 'true'))
+
+  - task: 1ESLighthouseEng.PipelineArtifactCaching.SaveCacheV1.SaveCache@1
+    inputs:
+      keyfile: '.yarnrc, remote/.yarnrc, **/yarn.lock, !**/node_modules/**/yarn.lock, !**/.*/**/yarn.lock, !samples/**/yarn.lock'
+      targetfolder: '**/node_modules, !**/node_modules/**/node_modules, !samples/**/node_modules'
+      vstsFeed: 'BuildCache'
+    condition: and(succeeded(), ne(variables['CacheRestored'], 'true'))
+
+  - script: |
+      set -e
+      yarn postinstall
+    displayName: Run postinstall scripts
+    condition: and(succeeded(), eq(variables['CacheRestored'], 'true'))
+
+  - script: |
+      set -e
+      yarn gulp mixin
+    displayName: Mix in quality
+
+  - script: |
+      set -e
+      yarn gulp vscode-darwin-min
+    displayName: Build
+
+  - script: |
+      set -e
+      yarn gulp install-sqltoolsservice
+    displayName: Install sqltoolsservice
+
+  - task: ArchiveFiles@2 # WHY ARE WE DOING THIS?
```

Comment:
We disabled the archive compiled files task a while back - this seems to fall in that same bucket so I see no reason to keep it around

Reply:
This wasn't disabled in our current task group for builds

Reply:
Yeah, these ones weren't. We disabled the package compiled files tasks. I see no reason to keep these ones around either though. 

![image](https://user-images.githubusercontent.com/28519865/68986004-fb105e00-07cf-11ea-884b-99fa085c4bd6.png)

Reply:
Karl recommended basically just keep what we have in the task groups, and then once these stabilize we can look at cleaning up these tasks. Hopefully I'm planning on focusing on this next week as part of dri.